README.md

```diff
@@ -3,3 +3,18 @@
 [WIP] An adapter to let you use [Twig](https://github.com/twigjs/twig.js) templates with [Fractal](http://github.com/frctl/fractal).
 
 Currently requires the (unreleased) Fractal v1.1.0-alpha.2 or greater - you can install it in your project using `npm i @frctl/fractal@next --save`.
+
+To install this adapter run this command:
+
+`npm install @frctl/twig`
+
+than open your fractial.js file and add following lines:
```

Comment:
Thanks for this!

I think it just needs the first "than" changing to "this". Also, it says `fractial.js` and it should be `fractal.js`.

Sorry for the delay in taking a look, we've just started getting together a team to help move Fractal forward, so we're becoming much more active with issues and PRs 😄